tests/ui/attributes/check-cfg_attr-ice.rs

```diff
@@ -0,0 +1,66 @@
+//! I missed a `cfg_attr` match in #128581, it should have had the same treatment as `cfg`.
+//!
+//! This test is not exhaustive, there's too many possible positions to check, instead it just does
+//! a basic smoke test in a few select positions to make sure we don't ICE for e.g.
+//! `#[cfg_attr::skip]`.
+//!
+//! issue: rust-lang/rust#128716
+#![crate_type = "lib"]
+
+#[cfg_attr::skip]
```

Comment:
I'm a bit confused by this. `cfg_attr::skip` is an invalid attribute, right? `cfg_attr` attributes are supposed to have the form `cfg_attr(cond, x)`.

Reply:
> I'm a bit confused by this. `cfg_attr::skip` is an invalid attribute, right? `cfg_attr` attributes are supposed to have the form `cfg_attr(cond, x)`.

Yes you are correct. When the attribute is of them form `#[cfg_attr::*]` however, it first fails in resolve, and doesn't reach the attribute checking passes yet.

Reply:
If so, can you change `skip` to something more obvious incorrect, such as `bogus` or `no_such_thing`?

Reply:
Sure thing, yeah `cfg_attr::skip` is an invalid attribute, it first emits an error in resolve and then fails in the check attr pass afterwards.

Reply:
Changed from `skip` -> `no_such_thing`.